examples/example_2.mff/po_videoSyncups.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="iso-8859-1" standalone="yes" ?>
<movieSyncUps>
    <movieSyncUpList>
        <syncUp>
            <sampleTime>556928</sampleTime>
            <movieTime>0.00000</movieTime>
            <category>0</category>
            <segment>0</segment>
            <movieDuration>0.00000</movieDuration>
        </syncUp>
        <syncUp>
            <sampleTime>571930</sampleTime>
            <movieTime>12.12613</movieTime>
            <category>0</category>
            <segment>0</segment>
            <movieDuration>0.00000</movieDuration>
        </syncUp>
        <syncUp>
            <sampleTime>2371968</sampleTime>
            <movieTime>1812.14514</movieTime>
            <category>0</category>
            <segment>0</segment>
            <movieDuration>0.00000</movieDuration>
        </syncUp>
        <syncUp>
            <sampleTime>4171961</sampleTime>
            <movieTime>3612.16577</movieTime>
            <category>0</category>
            <segment>0</segment>
            <movieDuration>0.00000</movieDuration>
        </syncUp>
        <syncUp>
            <sampleTime>5972005</sampleTime>
            <movieTime>5412.18799</movieTime>
            <category>0</category>
            <segment>0</segment>
            <movieDuration>0.00000</movieDuration>
        </syncUp>
        <syncUp>
            <sampleTime>6326153</sampleTime>
            <movieTime>5766.36670</movieTime>
            <category>0</category>
            <segment>0</segment>
            <movieDuration>0.00000</movieDuration>
        </syncUp>
    </movieSyncUpList>
</movieSyncUps>

```

Comment:
Ditto from my MovieSyncs comment.  Not sure if this `xml` is directly hand in hand with movieSyncs - we may have some poking around to do to incorporate this aspect of MFF.